pkg/controllers/jobset_controller.go

```diff
@@ -90,6 +90,11 @@ func (r *JobSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	if manager, found := js.Labels[jobset.LabelManagedBy]; found && manager != jobset.JobSetManager {
+		// the JobSet is not managed by this controller
```

Comment:
Do you think a log or event should be put here? This seems to be a bit difficult to debug if you weren't aware of this feature?

Reply:
Maybe a log ..... but it could get very chatty, so V5 maybe.